File: app/src/main/java/com/onix/streamer/demo/actioncamera/ActionCamStreamActivity.java
package com.onix.streamer.demo.actioncamera;

import android.os.Bundle;
import android.support.v4.app.FragmentManager;
import android.support.v7.app.AppCompatActivity;

import com.onix.streamer.demo.DemoActivity;
import com.onix.streamer.demo.R;

public class ActionCamStreamActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_streamer);

        String tag = ActionCamStreamFragment.class.getSimpleName();
        FragmentManager fm = getSupportFragmentManager();

        if (fm.findFragmentByTag(tag) == null) {
            getSupportFragmentManager().beginTransaction().replace(R.id.fragment_container,
                    ActionCamStreamFragment.newInstance(getIntent().getStringExtra(DemoActivity.ARG_SERVER_URL)), tag).commitAllowingStateLoss();
        }
    }
}

File: app/src/main/java/com/onix/avlib/demo/action_camera/ActionCamStreamActivity.kt
package com.onix.avlib.demo.action_camera

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.onix.avlib.demo.ARG_SERVER_URL
import com.onix.avlib.demo.R
import com.onix.avlib.demo.util.ext.argument

class ActionCamStreamActivity : AppCompatActivity() {

    private val serverUrl by argument<String>(ARG_SERVER_URL)

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_streamer)

        val tag = ActionCamStreamFragment::class.java.simpleName

        if (supportFragmentManager.findFragmentByTag(tag) == null) {
            supportFragmentManager.beginTransaction().replace(
                R.id.fragmentContainer, ActionCamStreamFragment.instance(serverUrl), tag
            ).commitAllowingStateLoss()
        }
    }

}
